### ISSUE TITLE
Datasets

### ISSUE THREAD

**Issue by saskra** [tianrun-chen/SAM-Adapter-PyTorch#11]
The "CHAMELEON Database - Animal Camouflage Analysis" appears to be unavailable: https://www.polsl.pl/rau6/pl/dataset/63-animal-camouflage-analysis . Do you know of an alternative link?

https://github.com/tianrun-chen/SAM-Adapter-PyTorch/blob/fb8349f9fdb94393838cbaf494f0a008bca0208c/README.md?plain=1#L20 

Which of the data sets exactly is meant here, or should/can you pack them all together? Is there perhaps a small toy dataset to test the general runnability?

**Comment by tianrun-chen** [comment]
Hello! You can access the CHAMELEON data at: https://www.polsl.pl/rau6/chameleon-database-animal-camouflage-analysis/, we have updated the link! The data means the dataset you use, you can use your customized data here! In our experiment, there is a folder for masks that contains the mask files, and another folder for images that stores the corresponding RGB images (it is important to verify the file names in the dataloader prior to training). The matching of masks and RGB images is based on their shared file names, which are usually numbered from 1 to N, where N represents the total number of images.

**Comment by saskra** [comment]
Thank you for the explanation. Unfortunately, the new link doesn't work either, but then I'll just test it with one of the other data sets first.

**Comment by tianrun-chen** [comment]
If you still cannot open the link, please can you kindly check this repo: https://github.com/DengPingFan/SINet, which have the dataset in Google Drive and Baidu YunPan.

**Comment by saskra** [comment]
Thank you, that worked!

You don't happen to have direct download links for wget or curl in the terminal, so that I can test it more easily on a graphics-free server? As it is, my local 16GB graphics card is not sufficient for this. Or have I overlooked settings besides the batch size that I could use to lower the memory requirements?
